[CUBRIDMAN-270] [KO][EN] Add a description of the restrictions for specifying functions or stored procedures as default values (#639)
CUBRID/cubrid-manual: en/sql/schema/table_stmt.rst

| TO_CHAR(number[, format])     | STRING        |
+-------------------------------+---------------+

.. note::

>     When using a function not specified in the table above as a **DEFAULT** value, it is evaluated at the time of table creation and stores the result executed with constant arguments.

Reviewer: ```suggestion
    When using a function not specified in the table above as a **DEFAULT** value, it is evaluated at the time of table creation and stored as the **DEFAULT** value.
```

>     Therefore, the default value for data **INSERT**\ ed into the column is the result stored at the time of table creation.

Reviewer: ```suggestion
    Therefore, the default value for data **INSERT**\ ed into the column is not the value of the function performed at **INSERT**, but the result value performed at the time of table creation.
```

>     Therefore, the default value for data **INSERT**\ ed into the column is not the value of the function performed at **INSERT**, but the result value performed at the time of table creation.

Reviewer 2: ```suggestion
    Therefore, during an **INSERT**, the default value used for the column is not the function result at the time of **INSERT**, but the value evaluated at the time the table was created.
```

>     In the case of stored functions, an error is returned because they cannot be pre-evaluated with constant arguments.

Reviewer: ```suggestion
    In the case of stored functions, it cannot be used as a **DEFAULT** value, so it returns an error.
```

>     In the case of stored functions, it cannot be used as a **DEFAULT** value, so it returns an error.

Reviewer 2: ```suggestion
    In the case of stored functions, they cannot be used as a **DEFAULT** value, and doing so will return an error.
```

>     In the case of stored functions, it cannot be used as a **DEFAULT** value, so it returns an error.

Reviewer 2: ```suggestion
    Stored functions cannot be used as **DEFAULT** values, and doing so results in an error.
```

.. code-block:: sql

    CREATE TABLE default_val_func_test (id INT DEFAULT RAND ());
    INSERT INTO default_val_func_test (id) VALUES (1);
    INSERT INTO default_val_func_test (id) VALUES (DEFAULT);
    INSERT INTO default_val_func_test (id) VALUES (DEFAULT);
    
    SELECT * FROM default_val_func_test;

::

               id
    =============

>             1
>        1572882525
>        1572882525

Author: ```suggestion
                1
       1572882525
       1572882525
```
